src/index.html

```diff
@@ -33,7 +33,7 @@
                 <div class="sidetab-top">
                     <img src="assets/svg/search_tab.svg" class="top-icon" />
                     <input class="sidetab" type="text" maxlength="20" placeholder="Enter stock name or symbol"
-                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" />
+                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" ng-keyup="searchCtrl.onSearchKeyUp($event)" />
                 </div>
```

Comment:
Not sure how I feel about this event being on keyup.

It's a bit weird not having the stock change on key down, and also if you hold down the key the selection only changes by one, and won't move throughout the search results, as it would on a keypress I believe.